Fix panic when message updates

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x60 pc=0x74cce9]

goroutine 1 [running]:
github.com/go-telegram-bot-api/telegram-bot-api.(*Message).IsCommand(...)
	go/pkg/mod/github.com/go-telegram-bot-api/telegram-bot-api@v4.6.4+incompatible/types.go:181
dutybot/internal/bot.StartBot()
	dutybot/internal/bot/bot.go:28 +0x2a9
main.main()
	dutybot/main.go:14 +0x20
exit status 2
```

--- a/internal/bot/bot.go
+++ b/internal/bot/bot.go
@@ -25,14 +25,22 @@ func StartBot() {
 	initHandlers()
 
 	for update := range updates {
-		if update.Message.IsCommand() {
-			go processCommands(bot, update.Message)
+		var msg *tgbot.Message
+
+		if update.EditedMessage != nil {
+			msg = update.EditedMessage
+		} else {
+			msg = update.Message
+		}
+
+		if msg.IsCommand() {
+			go processCommands(bot, msg)
 		} else {
 			sticker := tgbot.NewStickerShare(
-				update.Message.Chat.ID,
+				msg.Chat.ID,
 				"CAACAgIAAxkBAAM3X2xZtzvEBDmu4zcuRYYN8xW7hskAAqsAA5XcYhplKvU6wxFPMRsE",
 			)
-			sticker.ReplyToMessageID = update.Message.MessageID
+			sticker.ReplyToMessageID = msg.MessageID
 			_, err := bot.Send(sticker)
 			if err != nil {
 				return

